common/models/user.js

```diff
@@ -672,13 +672,18 @@ module.exports = function(User) {
       return process.nextTick(cb);
 
     var AccessToken = accessTokenRelation.modelTo;
-
     var query = {userId: {inq: userIds}};
     var tokenPK = AccessToken.definition.idName() || 'id';
     if (options.accessToken && tokenPK in options.accessToken) {
       query[tokenPK] = {neq: options.accessToken[tokenPK]};
     }
-
+    // add principalType in AccessToken.query if using polymorphic relations
+    // between AccessToken and User
+    var isTokenRelationToUserPolymorphic = AccessToken.relations.user.polymorphic &&
```

Comment:
Please store `AccessToken.relations.user` in a helper variable to avoid repetition and long lines.